ui/src/components/auth/LoginContainer.js

```diff
@@ -79,108 +63,139 @@ const LoginContainer = (props) => {
   };
 
   const inputBoxStyle = {
-    background: "#ffffff",
-    border: "1px solid #000000",
-    boxSizing: "border-box",
-    borderRadius: "6px",
-    padding: "3px 10px",
-    width: "250px",
-    color: "black",
-    paddingLeft: "5px",
+    color: 'black',
+    background: '#EDF2F7',
+    border: '1px solid #E2E8F0',
+    boxSizing: 'border-box',
+    borderRadius: '12px',
+    flex: 'none',
+    flexGrow: '0',
+    margin: '5px 0px',
+    padding: '15px',
+    fontSize: 'inherit',
+    fontWeight: '500',
   };
 
   const submitBoxStyle = {
-    justifyContent: "center",
-    alignItems: "center",
-    padding: "3px 19px",
-    background: "rgba(0, 0, 0, 0.04)",
-    border: "1px solid #FFFFFF",
-    boxSizing: "border-box",
-    backdropFilter: "blur(6px)",
-    borderRadius: "10px",
-    width: "250px",
-    fontWeight: "bold",
-  };
-
-  const signupBoxStyle = {
-    justifyContent: "center",
-    alignItems: "center",
-    padding: "3px 19px",
-    background: "rgba(0, 0, 0, 0.04)",
-    border: "1px solid #FFFFFF",
-    boxSizing: "border-box",
-    backdropFilter: "blur(6px)",
-    borderRadius: "10px",
-    width: "300px",
-    fontWeight: "bold",
+    color: 'white',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignSelf: 'stretch',
+    padding: '18px',
+    boxSizing: 'border-box',
+    backdropFilter: 'blur(6px)',
+    borderRadius: '12px',
+    fontWeight: 'bold',
+    fontSize: 'inherit',
+    width: '100%',
   };
 
   return (
     <>
-      <Stack background="#2C5282" color="white" padding="20px">
-        <Flex>
-          <Box>
-            <Link href="../">
-              <ChevronLeftIcon boxSize={10} />
-              Return to hongkong.hmcc.net
+      <Box
+        minH="100vh"
+        w="full"
+        bgImage={`url(${
+          process.env.PUBLIC_URL + '/images/default-hk-background.jpg'
+        })`}
+        bgRepeat="no-repeat"
+        bgAttachment="fixed"
+        bgSize="cover"
+      >
+        <Flex
+          flexDirection="column"
+          align="center"
+          w={['100vw', '85vw', '70vw', '60vw', '40vw']}
+          m="auto"
+          p="10"
+        >
+          <Box width="full" alignSelf="flex-start" mb="5" color="#FFF">
```

Comment:
small thing but i think this should only be rendered in `/signup/form` according to the figma. It also might be misleading since on `/login` or `/signup` it just redirects people to `/` or the homepage instead of whatever page they were on